site/en/docs/devtools/css/reference/index.md

```diff
@@ -289,6 +289,53 @@ The Coverage tab shows you what CSS a page actually uses.
 
 See [Force DevTools Into Print Preview Mode][11].
 
+## Copy CSS {: #copy-css }
+
+From a single drop-down menu in the **Styles** pane, you can copy separate [CSS rules, declarations, properties, values](https://www.w3schools.com/css/css_syntax.asp), and more.
```

Comment:
Avoid using W3 Schools links, in preference for MDN or our own material on web.dev.

In this case https://developer.mozilla.org/docs/Learn/CSS/First_steps/What_is_CSS#css_syntax